odonto-schedulle-service/src/main/resources/db/migration/V1.1__CreateTables.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE `status` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `descricao` varchar(30) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `UKY_descricao` (`descricao`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+
+CREATE TABLE `agendamento` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `client_id` bigint NOT NULL,
+  `created_at` datetime(6) NOT NULL,
```

Comment:
Este campo deveria gravar a datahora automaticamente, substituir por 
`created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP`